src/index.html

```diff
@@ -2,37 +2,58 @@
 layout: pageNoEdit
 permalink: /
 ---
+<div class="px-4 py-5 my-5 text-center" style="padding-bottom: 0 !important;">
+  <img class="d-block mx-auto mb-4 border-0" src="{{site.baseurl}}/assets/images/pillar.svg" alt="" width="72" width="75" style="border: none;">
```

Comment:
Image should have `alt` tag for accessability. Please use extention https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd to help determin accessability.